admin/subjects/views.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.core.urlresolvers import reverse_lazy
+from django.views.generic import ListView, UpdateView
+
+from admin.subjects.forms import SubjectForm
+from osf.models.subject import Subject
+from osf.models.preprint_provider import PreprintProvider
+
+
+class SubjectListView(PermissionRequiredMixin, ListView):
+    model = Subject
+    permission_required = 'osf.view_subject'
+    paginate_by = 100
+    raise_exception = True
+
+    def get_queryset(self):
+        request = self.request.GET
+        qs = super(SubjectListView, self).get_queryset().order_by('text')
```

Comment:
Currently, there are about 1200, and that will only grow with additional "custom taxonomies"